sf-docker/all-in-one/create-wl-stream-generation.sh

```diff
@@ -1,10 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 
-source .env
+getvalue() {
+    local key="$1"
+    local value=$(grep -E ^${key}= .env | cut -d '=' -f2- | cut -d$'\r' -f1)
```

Comment:
there are some very minor differences between this function and the grep from .env in `run.sh` script, why is that?

also maybe we can reuse the function in both scripts

Reply:
This version can handle values containing char '=', the run.sh one can't handle them (thus problems with connection string).
I'll fix the run.sh one.